frontend/src/css/modules/challenge-create.scss

```diff
@@ -4,3 +4,7 @@
 	margin-bottom: 20px;
 	margin-left: 11px;
 }
+
+.add-margin {
```

Comment:
@RishabhJain2018 please follow proper naming for classes, add-margin is way too common.